Modules/Core/Common/test/itkCommonTypeTraitsGTest.cxx

```diff
@@ -84,3 +84,30 @@ TEST(CommonTypeTraits, ContinuousIndexIsPOD) {
   EXPECT_EQ(std::is_trivial<T>::value, true);
   EXPECT_EQ(std::is_standard_layout<T>::value, true);
 }
+
+/************ FixedArray: noexcept move checks *************/
+
+/* Dummy class without noexcept move contructors. */
+struct MightThrowAtMove
+{
+    MightThrowAtMove() {}
+    MightThrowAtMove(MightThrowAtMove&&) {}
```

Comment:
Clearly this `MightThrowAtMove` move-constructor of does not throw at all! You _might_ consider adding something like `if (std::rand() == 42) throw std::exception{}` to the body of this move-constructor.

Reply:
Maybe change the name to: `NotNoexceptMove` ? We don't need to actually throw anything for the type traits checking. What do you think?

Reply:
Changed name to NotNoExceptMove